frontend/public/components/monitoring/targets.tsx

```diff
@@ -1,26 +1,32 @@
-import * as _ from 'lodash-es';
 import {
   GreenCheckCircleIcon,
+  ListPageBody,
   PrometheusEndpoint,
   RedExclamationCircleIcon,
   RowFilter,
+  RowProps,
+  TableColumn,
 } from '@console/dynamic-plugin-sdk';
-import { Alert, Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+import {
+  ListPageFilter,
+  ListPageHeader,
+  ResourceLink,
+  Timestamp,
+  useK8sWatchResource,
+  useListPageFilter,
+  VirtualizedTable,
+} from '@console/dynamic-plugin-sdk/src/lib-core';
+import { Alert, Breadcrumb, BreadcrumbItem, Tooltip } from '@patternfly/react-core';
 import { sortable } from '@patternfly/react-table';
+import * as fuzzy from 'fuzzysearch';
+import * as _ from 'lodash-es';
```

Comment:
Maybe it would make sense to import only the functions needed here to allow tree shaking

Reply:
👍 

Console is using `import * as _ from 'lodash-es'` everywhere else, but I think it makes sense to change it in our case since we are refactoring to be a plugin.

In a few places throughout the monitoring page, there's really no advantage to using Lodash over native JavaScript, but I will leave refactoring them away for future PRs.